SampleUserControlLibrary/SubscriptionKeyPage.xaml.cs

```diff
@@ -223,6 +222,16 @@ private void SaveSubscriptionEndpointToIsolatedStorage(string subscriptionEndpoi
         }
 
         /// <summary>
+        /// Set a default endpoint when there is no legal endpoint value
+        /// </summary>
+        /// <param name="msg"></param>
+        public void setDefaultSubscriptionEndpoint(string msg){
```

Comment:
1) Drop the `default` in function name and comments.
2) The variable name `msg` is obscure, let us make it `endpoint` instead.
3) Function name should follows PascalCase.